adobe_vipm/flows/helpers.py

```diff
@@ -263,14 +303,60 @@ def get_licenses_and_consumables_count(subscriptions):
         """
         count_licenses = 0
         count_consumables = 0
-        for subscription in subscriptions["items"]:
-            if is_consumables_sku(subscription["offerId"]):
-                count_consumables += subscription["currentQuantity"]
+
+        if not subscriptions or "items" not in subscriptions:
```

Comment:
If it is possible to extract list of subscriptions somewhere before or even in adobe client would be cool, cause you are going to remove this checks for 'Items' across all the ocde